flower_classifier/train.py

```diff
@@ -14,6 +15,14 @@
 @hydra.main(config_path="../conf", config_name="config")
 def train(cfg):
     data_module = hydra.utils.instantiate(cfg.dataset)
+    data_module.prepare_data()
```

Comment:
i don't love the idea of reaching in to manually invoke this, but can't think of a better option at the moment